CITATION.cff

```diff
@@ -1,290 +1,431 @@
-# YAML 1.2
 # Metadata for citation of this software according to the CFF format (https://citation-file-format.github.io/)
 cff-version: 1.2.0
-message: This research made use of Gammapy, a community-developed core Python package
-  for gamma-ray astronomy. https://ui.adsabs.harvard.edu/abs/2017ICRC...35..766D/abstract
 title: 'Gammapy: Python toolbox for gamma-ray astronomy'
 abstract: 'Gammapy analyzes gamma-ray data and creates sky images, spectra and lightcurves,
   from event lists and instrument response information; it can also determine the
   position, morphology and spectra of gamma-ray sources. It is used to analyze data
-  from H.E.S.S., Fermi-LAT, and the Cherenkov Telescope Array (CTA).'
+  from H.E.S.S., Fermi-LAT, HAWC, and the Cherenkov Telescope Array (CTA).'
+type: software
+message: "If you use this software, please cite it using the metadata from this file."
+url: "https://gammapy.org/"
 doi: 10.5281/zenodo.4701488
+preferred-citation:
+  authors:
+  - family-names: "Deil"
+    given-names: "Christoph"
+  - family-names: "et al."
+  title: "Gammapy - A prototype for the CTA science tools"
+  doi: 10.48550/ARXIV.1709.01751
+  url: "https://arxiv.org/abs/1709.01751"
+  type: conference-paper
+  year: 2017
+  conference:
+    name: "Proc. 35th ICRC, Busan, South Korea, PoS(ICRC2017)766"
+  publisher:
+    name: "arXiv"
+keywords:
+- Astronomy
+- "Gamma-rays"
+- "Data analysis"
+version: "1.0"
+date-released: 2022-09-15
+repository-code: "https://github.com/gammapy/gammapy"
+license: BSD-3-Clause
+contact:
+- email: GAMMAPY-COORDINATION-L@IN2P3.FR
+  name: "Coordination committee of the Gammapy project"
 authors:
-- given-names: Axel
-  family-names: Donath
-  affiliation: Center for Astrophysics | Harvard & Smithsonian, CfA, 60 Garden St.,
-    02138 Cambridge MA, US
+- family-names: Acero
+  given-names: Fabio
+  email: fabio.acero@cea.fr
+  orcid: https://orcid.org/0000-0002-6606-2816
+  affiliation: AIM (DAp)
```

Comment:
Hi @bkhelifi ,
Could you update my affiliation to the following one:
Université Paris-Saclay, Université Paris Cité, CEA, CNRS, AIM de Paris-Saclay, 91191 Gif sur Yvette

and the city to Gif sur Yvette
ORCID number is correct.
thanks !